docs/zh_cn/get_started.md

```diff
@@ -226,12 +223,15 @@ result = inference_model(
 你可以直接运行预编译包中的 demo 程序，输入 SDK Model 和图像，进行推理，并查看推理结果。
 
 ```shell
-cd mmdeploy-1.0.0rc3-linux-x86_64-cuda11.1-tensorrt8.2.3.0
+wget https://github.com/open-mmlab/mmdeploy/releases/download/v1.0.0rc3/mmdeploy-1.0.0rc3-linux-x86_64-cuda11.3.tar.gz
+tar xf mmdeploy-1.0.0rc3-linux-x86_64-cuda11.3
```

Comment:
mmdeploy-1.0.0rc3-linux-x86_64-cuda11.3.tar.gz?

Reply:
do you mean the tensorrt suffix? I remove it as this package support both tensorrt and onnxruntime-gpu inference.

Reply:
OK